nixos/modules/installer/cd-dvd/installation-cd-graphical-kde.nix

```diff
@@ -30,15 +30,27 @@ with lib;
       Version=1.0
       Type=Application
       Name=NixOS Manual
-      Exec=firefox ${config.system.build.manual.manualHTMLIndex}
```

Comment:
Just realized that `nixos-manual.desktop` generation isn't needed anymore,
as `nixos-manual` has a desktop item that will intelligently launch the default browser.

I think I'll fix this in the second pr, dropping this bit.

(not the gparted part though)

Reply:
So this can be removed?